contracts/Cargo.toml

```diff
@@ -14,14 +14,15 @@ alloy-sol-types.workspace = true
 alloy-sol-macro.workspace = true
 alloy-sol-macro-expander.workspace = true
 alloy-sol-macro-input.workspace = true
-stylus-sdk.workspace = true
+stylus-sdk = { workspace = true, default-features = true }
 keccak-const.workspace = true
 openzeppelin-stylus-proc.workspace = true
 
 [dev-dependencies]
 alloy-primitives = { workspace = true, features = ["arbitrary"] }
 motsu.workspace = true
 rand.workspace = true
+stylus-sdk.workspace = true
```

Comment:
stylus-sdk is listed both in `dependencies` and in `dev-dependencies`

Reply:
Since OL agreed to have a default feature for ENABLING hostio caching, now we should write `stylus-sdk = { version = "0.7.0", default-features = false }` on the workspace level, and `stylus-sdk = { workspace = true, default-features = true }` for `[dependencies]`.
And use without default features for unit tests `[dev-dependencies]`

Reply:
I would better add a feature for disabling caching (not enabling). It would more simple